zhaquirks/tuya/ts0601_pirmotion.py

```diff
@@ -0,0 +1,140 @@
+"""BlitzWolf IS-3/Tuya motion rechargeable occupancy sensor."""
```

Comment:
I think this comment should updated?

Reply:
I have updated this. Please let me know if you want any more changes.